### PR TITLE
[feat]: Added the button component

### PR DESCRIPTION
**Pull request by aryan-13**
- Added style sheet for the button component

Here is the preview: 
button

### CODE REVIEW

**Comment by stutisk** [comment]
looks  good to me !

**Review by abhijeetekad — APPROVED** [review]

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.btn {
+	text-transform: uppercase;
+	text-decoration: none;
+	padding: 1.2rem 2.5rem;
+	display: flex;
+	color: inherit;
+	border-radius: 1rem;
+	transition: all 0.2s;
+	position: relative;
+	/* font-size: inherit; */
```
Comment (paarthbajaj) [inline components/button.css:10]: No need to keep commented code btw.

File: documentation.html [hunk 2]
```diff
@@ -122,8 +122,163 @@ <h1 class="heading--1-medium">Code</h1>
...
+					<div class="main__example row-container">
+						<a href="" class="btn btn-primary">Button</a>
+						<a href="" class="btn btn-secondary">Button</a>
+						<a href="" class="btn btn-ghost">Button</a>
+						<a href="" class="btn btn-link">Button</a>
+					</div>
+					<h1 class="heading--1-medium">Code</h1>
+					<div class="main__code">
+						<code class="main__code-bg">
+							<iframe
```
Comment (paarthbajaj) [inline documentation.html:183]: If possible, avoid iframe. The reason is Iframes cause SEO problems, you can google the alternatives and can read more about them. Please correct me if I said something wrong.

File: style.css [hunk 3]
```diff
@@ -49,4 +49,4 @@
 .btn-secondary {
 	border: 0.2rem solid var(--color-primary);
 	color: var(--color-primary);
-}
+} */
```
Comment (paarthbajaj) [inline style.css:52]: Apart from my comments, everything looks good, and don't forget to delete the commented code. Kudos to the efforts. Well donee.

**Review by paarthbajaj — COMMENTED** [review]
Apart from my comment, everything looks good, and don't forget to delete the commented code. Kudos to the efforts. Well done.

**Review by dikshadhote — COMMENTED** [review]
Please format html file, the code doesn't seem readable

File: documentation.html (hunk 2 above)
Comment (aryan-13) [reply documentation.html:183]: I am using carbon for the code section and it gives the snippet for iframe itself> this is why I have to use the iframe.

**Comment by aryan-13** [comment]
> Please format html file, the code doesn't seem readable

I am using prettier for it, I'll try to fix it.

**Comment by aryan-13** [comment]
> Apart from my comment, everything looks good, and don't forget to delete the commented code. Kudos to the efforts. Well done.

Yeah Sure,  Thanks!

File: documentation.html (hunk 2 above)
Comment (paarthbajaj) [reply documentation.html:183]: Oh ok. Alright then.

**Comment by dikshadhote** [comment]
> > Please format html file, the code doesn't seem readable
> 
> I am using prettier for it, I'll try to fix it.

Sure!

**Pull request merged**